gateway/src/main/java/com/hurence/webapiservice/http/grafana/GrafanaApiImpl.java

```diff
@@ -116,6 +127,92 @@ at initialization (did not successfully reproduced this in a unit test).//TODO
                 }).subscribe();
     }
 
+    @Override
+    public void export(RoutingContext context) {
+        final long startRequest = System.currentTimeMillis();
+        final JsonObject requestBody;
```

Comment:
I do not see the purpose to keep the requestBody in the general scope. Indeed if we are parsing the requestBody in a Bean it is so that we can use that bean and add convenient method in this Bean. In this case the Bean is the class "QueryRequestParam"

Reply:
done